### DISCUSSION TITLE
How much period is for EWMA Very Long, Long, Medium & Short period

### DISCUSSION THREAD

**Discussion by nithi2023** [ctubio/Krypto-trading-bot#1093] [Q&A] [answered]
Hi, 
I want to know periods used for EWMA Very Long, Long, Medium & Short period and what is a 'period' actually mean?

My bot has `ewma price` enabled for a period 200 (default). 

the stats graph shows "EWMA Very long" as 58145  & My Ping and Pong width is `0.01`. but but actual placed bid price is 57947 only. any reason why?
 
Sorry, but I tried to find information online. I am finding it difficult to get answer for this particular question.

**Comment by ctubio** [comment, marked as answer]
:koala: the periods to calculate these stats are whatever periods you put in your own `verylong`, `long`, `medium` and `short` quoting parameters while using some `apMode = EWMA_*`
